plugins/inputs/gtfs/README.md

```diff
@@ -0,0 +1,81 @@
+# GTFS
+
+This input plugin can be used to gather vehicle positions, trip updates and
+service alerts from a GTFS-realtime feed. [GTFS-realtime](https://developers.google.com/transit/gtfs-realtime) 
+is an extension of the [General Transit Feed Specification](https://developers.google.com/transit/gtfs).
+
+### Configuration
+
+This plugin can be configured to collect any combination of vehicle positions, trip updates
+and service alerts. Enable any one of these three input types by populating its URL in the
+plugin configuration. At least one of the three must be configured.
+
+```toml
+[[inputs.gtfs]]
+  ## API Key
+  # key = "${GTFS_API_KEY}"
+  ## URL for fetching vehicle positions
+  # vehicle_positions_url = "https://host.test/VehiclePositions.pb"
+  ## URL for fetching vehicle positions
+  # trip_updates_url = "https://host.test/TripUpdates.pb"
+  ## URL for fetching vehicle positions
+  # service_alerts_url = "https://host.test/ServiceAlerts.pb"
+
+  ## Optional HTTP Basic Auth Credentials
+  # username = "username"
+  # password = "pa$$word"
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
+
+  ## Amount of time allowed to complete the HTTP request
+  # timeout = "5s"
+```
+
+### Metrics
+
+Metrics can be collected for each of three GTFS-realtime feed types: vehicle positions, trip updates and service alerts.
```

Comment:
Please name the output metrics here as a list in form
```
- gtfs_position
- gtfs_updates
- gtfs_alerts
```
so a user gets an idea what to expect.

Reply:
Will do.